trading_calendars/trading_calendar.py

```diff
@@ -224,8 +280,13 @@ def close_offset(self):
 
     @lazyval
     def _minutes_per_session(self):
-        diff = self.schedule.market_close - self.schedule.market_open
-        diff = diff.astype('timedelta64[m]')
+        close_to_open_diff = (
+            self.schedule.market_close - self.schedule.market_open
+        )
+        break_diff = (
+            self.schedule.break_end - self.schedule.break_start
+        ).fillna(pd.Timedelta(seconds=0))
```

Comment:
(Another case where I wonder if it would make sense to make break_end/break_start dense.)

Reply:
I'm not sure what you mean by make them dense? You could definitely have breaks that change over time (both over duration and whether they exist or not).

Reply:
Ah, nevermind. This makes sense now.